### PR TITLE
Feature/2489/subtitle for custom view modal

### PR DESCRIPTION
**Pull request by jannikr**
# Description to Custom Views modal

Issue: #2489

## Description

Before:
Screenshot 2022-10-11 at 11 37 28

Now:
Screenshot 2022-10-11 at 11 35 04

### CODE REVIEW

**Comment by jannikr** [comment]
@ce-bo We could think about removing the additional button for adding custom configs on the main page, because we already have the option in the current window (see screenshot). This would lead to more space in the toolbar and avoid redundant description texts.

File: visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigList.component.html [hunk 1]
```diff
@@ -9,12 +9,17 @@ <h3>Custom Views</h3>
 			<cc-add-custom-config-button class="custom-configs-button-in-custom-views"></cc-add-custom-config-button>
 		</div>
 	</div>
+	<div class="description">
+		Custom Views allow you to save and upload your individual configurations. Find out more about Custom Views in the
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigList.component.html:13]:
```suggestion
		Custom Views allow you to save and upload your individual configurations for certain maps. Find out more about Custom Views in the
```

File: visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigList.component.scss [hunk 2]
```diff
@@ -38,7 +42,8 @@
 	}
 
 	.no-custom-configs-box {
-		padding: 16px;
+		text-align: center;
+		padding: 50px 0 50px 0;
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigList.component.scss:46]:
```suggestion
		padding: 50px 0;
```

**Review by Hall-Ma — APPROVED** [review]
👍

File: gh-pages/_docs/06-04-custom-view.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+permalink: /docs/custom-views/
+title: "Custom View"
+---
+
+Custom Views allow you to save and upload your individual configurations for certain maps. To save new configurations there is a "shortcut-button". In order to be able to access more settings, there is another button next to the "shortcut-button" with the following functions:
```
Comment (ce-bo) [inline gh-pages/_docs/06-04-custom-view.md:6]: The user could ask "Where can I find the shortcut-button?".

File: gh-pages/_docs/06-04-custom-view.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+-   **Upload Custom Views**: Provides the possibility to upload previously downloaded Custom Views.
+-   **Download Custom View(s)**: If a Custom View has been created, this function offers the possibility to export the created Custom Views in order to share the configuration file.
+-   **Add Custom View**: For adding Custom Views. After a Custom View is added, the configurations are saved in the browser.
```
Comment (ce-bo) [inline gh-pages/_docs/06-04-custom-view.md:10]: Vorschlag: ```suggestion
-   **Add Custom View**: For adding Custom Views. After a Custom View is added, the configurations are saved in the browser. Then, you can reapply it to restore all the settings that are saved in the Custom View.
```

**Review by ce-bo — APPROVED** [review]
LGTM: I added some remarks though to clearify some points.

File: gh-pages/_docs/06-04-custom-view.md (hunk 3 above)
Comment (jannikr) [reply gh-pages/_docs/06-04-custom-view.md:6]: I would suggest adapting the [user control documentation](https://maibornwolff.github.io/codecharta/docs/user-controls/) soon and then referring to it. That would have taken too much time now. I would be happy to open an issue for this @ce-bo ?

Comment (ce-bo) [reply gh-pages/_docs/06-04-custom-view.md:6]: Perfect, thank you!

**Pull request merged**